packages/runtime/package.json

```diff
@@ -17,7 +17,7 @@
     "reset-rn": "watchman watch-del-all; rm -rf $TMPDIR/metro-cache-*; rm -rf $TMPDIR/haste-map-*; rm -rf $TMPDIR/metro-symbolicate*",
     "reset-yarn": "yarn cache clean",
     "check-licenses": "yarn licenses list --prod | grep '\\(─ GPL\\|─ (GPL-[1-9]\\.[0-9]\\+ OR GPL-[1-9]\\.[0-9]\\+)\\)' && echo 'Found GPL license(s). Use 'yarn licenses list --prod' to look up the offending package' && exit 1 || echo 'No GPL licenses found'",
-    "build": "yarn run build:ts && yarn run build:metro",
+    "build": "yarn run build:ts",
```

Comment:
Removed the `build:metro` part as it currently fails, since `react-native` is installed in the root `node_modules` in this monorepo setup.

We'll have an alternative check once we had a test app to this monorepo.